scripts/helpers.py

```diff
@@ -622,26 +622,22 @@ def get_conv_factors(sector):
             "Gas Coke": 0.007326,
             "Refinery gas": 0.01375,
             "Coal Tar": 0.007778,
+            "Paraffin waxes": 0.01117,
+            "Ethane": 0.01289,
+            "Oil shale": 0.00247,
         }
     return fuels_conv_toTWh
 
 
 def aggregate_fuels(sector):
     gas_fuels = [
         "Blast Furnace Gas",
-        "Biogases",
```

Comment:
Not sure if it should be done in this PR, but all these entries gas_fuels/biomass_fuels, etc may be better stored as a dictionary and that could help automatize and clean this script.
Example:
```
fuels_conv = {
"gas": [ ....],
"biomass": [...],
etc
}
```
Not sure if now is the best moment to address this

Reply:
Possible of course.

Reply:
Moved to helpers